mypy/types.py

```diff
@@ -384,6 +384,7 @@ class Instance(Type):
 
     def __init__(self, typ: Optional[mypy.nodes.TypeInfo], args: List[Type],
                  line: int = -1, column: int = -1, erased: bool = False) -> None:
+        assert(typ is None or typ.fullname() not in ["builtins.Any", "typing.Any"])
```

Comment:
Ah I figured that out myself, but my comment was lost somehow. What about `builtins.Any` and `typing.Any`? Why are we creating an `Instance` type that corresponds to `Any`, and why is there a `builtins.Any` in addition to `typing.Any`? Shouldn't we use `AnyType`?